docs/source/using/managing-minishift.adoc

```diff
@@ -53,6 +53,32 @@ the GitHub issue https://github.com/minishift/minishift/issues/179[#179].
 The link:../command-ref/minishift_delete{outfilesuffix}[`minishift delete`] command deletes the OpenShift cluster,
 and also shuts down and deletes the Minishift VM. No data or state are preserved.
 
+[[choosing-vm]]
+== Choosing the VM image
+
+When you start Minishift, it downloads an launches a VM image that runs the OpenShift cluster.
+The VM image is a Live ISO that contains an operating system as well as all the dependencies to
+run the OpenShift cluster.
+
+The following VM images are available:
+
+- link:https://github.com/minishift/minishift-b2d-iso[Boot2Docker] (Default). This image is
```

Comment:
"Boot2Docker ISO" and "CentOS ISO", especially in the later case, just having CentOS standalone seems confusing to me.